setup.cfg

```diff
@@ -27,7 +27,7 @@ classifiers =
 [options]
 packages = find:
 install_requires =
-	apns2>=0.3.0
+	apns2>=0.3.0,<0.6.0
```

Comment:
0.6.0 is the latest version, shouldn't this be >=0.6.0,<0.7?

Reply:
Hmm. Not sure if I understand you.

`0.6.0` is the version the doesn't work (because of the changes),  so using `>=0.6.0,<0.7` will imply that we support `0.6.0`, or am I wrong?